virttest/nfs.py

```diff
@@ -285,8 +285,9 @@ def is_mounted(self):
                                                                 self.mount_dir)
         cmd = self.ssh_cmd + "'%s'" % find_mountpoint_cmd
         logging.debug("The command: %s", cmd)
-        if process.system(cmd, verbose=True, ignore_status=True, shell=True):
-            return False
+        status, output = process.getstatusoutput(cmd)
+        if status:
+            logging.debug("The command result: <%s:%s>", status, output)
```

Comment:
Why change the logical here? No need to return False?

Reply:
Fixed